db/migrate/20220915002454_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Same here, Please edit this migration because now it's creating a foreign key as `user_id` and It's supposed to be `author_id` like the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
Thank you, I have fixed it. Your hint was helpful.